packages/terra-spacer/src/Spacer.scss

```diff
@@ -0,0 +1,335 @@
+$small-minus-2: 0.2142857143rem;
```

Comment:
You should probably just make these sizes the themeable variables instead of each of the different style permutations.